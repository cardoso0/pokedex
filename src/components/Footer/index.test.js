import { render, screen } from "@testing-library/react"
import renderer from 'react-test-renderer'

import { Footer } from "."

describe("FooterComponent", () => {
  it ("component rendering footer", () => {
    const tree = renderer
    .create(<Footer />)
    expect(tree).toBeTruthy()
  })

  it ("must contain this paragraph Desenvolvido por Felipe Cardoso", () => {
    render(<Footer />)
    const footerParagraph = screen.getByText("Footer.copy")
    expect(footerParagraph).toBeTruthy()
  })

  it ("component rendering", () => {
    const tree = renderer
    .create(<Footer />)
    .toJSON()
    expect(tree).toMatchSnapshot()
  })
})