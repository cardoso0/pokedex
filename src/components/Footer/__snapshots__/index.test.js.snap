// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component rendering 1`] = `
<footer
  className="sc-bczRLJ xgqim"
>
  <main
    className="sc-gsnTZi VpOde"
  >
    <p>
      Desenvolvido por Felipe Cardoso
    </p>
  </main>
</footer>
`;
